code-studio/package.json

```diff
@@ -60,18 +62,22 @@
     "eslint": "^2.3.0",
     "factor-bundle": "^2.5.0",
     "gaze": "^1.0.0",
+    "history": "^2.0.1",
     "http-server": "^0.9.0",
     "istanbul": "^0.4.2",
     "johnny-five": "bcjordan/johnny-five#with-firmata-fix",
     "mkdirp": "^0.5.1",
     "mocha": "^2.3.4",
     "mochify": "^2.15.0",
     "mochify-istanbul": "^2.4.1",
+    "moment": "2.12.0",
     "node-sass": "^3.4.2",
     "node-watch": "^0.3.5",
     "object-assign": "^4.0.1",
     "phantomjs-prebuilt": "^2.1.3",
     "playground-io": "bcjordan/playground-io#tap-cap-touch",
+    "react-bootstrap": "^0.28.5",
+    "react-router": "^2.0.1",
     "recursive-readdir-sync": "^1.0.6",
```

Comment:
anything that gets used in the front end should probably be part of the `dependencies` section. I don't think it will make a difference the way we have things set up, but "best practices" and all that.